src/SerilogTracing/Instrumentation/ActivityInstrumentation.cs

```diff
@@ -248,6 +254,199 @@ class TextException(
         public override string ToString() => toString ?? "No information available.";
     }
 
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <param name="configureReplacement"></param>
+    /// <param name="postSamplingFilter"></param>
+    /// <param name="inheritTags"></param>
+    /// <param name="inheritParent"></param>
+    /// <param name="inheritFlags"></param>
+    /// <param name="inheritBaggage"></param>
+    /// <returns></returns>
+    public static void StartReplacementActivity(
+        Func<Activity?, bool> postSamplingFilter,
+        Action<Activity> configureReplacement,
+        bool inheritTags = true,
```

Comment:
We may want to introduce an options type for this. The `IncomingTraceparent` enum is nice, but for this API it might be nicer to have more control over what gets carried into the replacement